recipes-core/images/yoe-simple-image.bb

```diff
@@ -33,24 +33,24 @@ UPDATE_IMAGES_sama5d27-som1-ek-sd ?= "\
 UPDATE_IMAGES_ ?= "${UPDATE_IMAGES} update.sha256"
 KERNEL_IMAGE ?= "zImage-initramfs-${MACHINE}.bin"
 
-VERSION ?= "0.00.00"
+IMG_VERSION ?= "0.0.0"
```

Comment:
its already set in site.conf do we need to re-assign it here ?

Reply:
I was simply modifying what was there, so did not think about it. Seems it would be nice to have a default, so stuff does not break if use does not have it set in site.conf.